tests/quantization/modelopt/test_modelopt.py

```diff
@@ -0,0 +1,323 @@
+import gc
+import tempfile
+import unittest
+
+from diffusers import SanaPipeline, SanaTransformer2DModel, NVIDIAModelOptConfig
+from diffusers.utils import is_nvidia_modelopt_available, is_torch_available
+from diffusers.utils.testing_utils import (
+    backend_empty_cache,
+    backend_reset_peak_memory_stats,
+    enable_full_determinism,
+    nightly,
+    numpy_cosine_similarity_distance,
+    require_accelerate,
+    require_big_accelerator,
+    require_torch_cuda_compatibility,
+    torch_device,
+)
+
+if is_nvidia_modelopt_available():
+    import modelopt.torch.quantization as mtq
+
+if is_torch_available():
+    import torch
+    from ..utils import LoRALayer, get_memory_consumption_stat
+
+enable_full_determinism()
+
+
+@nightly
+@require_big_accelerator
+@require_accelerate
+class ModelOptBaseTesterMixin:
+    model_id = "Efficient-Large-Model/Sana_600M_1024px_diffusers"
```

Comment:
If we can use smaller checkpoints like `torchao` does in its tests, let's try that.